src/main/kotlin/org/jitsi/nlj/rtp/bandwidthestimation/GoogleCcEstimatorConfig.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.nlj.rtp.bandwidthestimation
+
+import org.jitsi.config.JitsiConfig
+import org.jitsi.metaconfig.config
+import org.jitsi.nlj.util.Bandwidth
+import org.jitsi.nlj.util.kbps
+import org.jitsi.nlj.util.mbps
+
+private val defaultMinBw: Bandwidth = 30.kbps
+private val defaultMaxBw: Bandwidth = 20.mbps
+
+class GoogleCcEstimatorBweConfig {
+    companion object {
+        val minBw: Bandwidth by config {
+            "jmt.bwe.google-cc.min-bw".from(JitsiConfig.newConfig)
+                .convertFrom<String> { Bandwidth.fromString(it) }
+            "default" { defaultMinBw }
```

Comment:
Please remove this and the one below, we can assume that `reference.conf` has the required defaults (we'll fail with an exception if it doesn't)